website/content/docs/projects/remote.mdx

```diff
@@ -60,7 +60,7 @@ logic is executing remotely.
 
 Waypoint only allows a single operation per distinct `(workspace, project, app)`
 combination. This applies to both local and remote operations. Therefore,
-if one user runs `waypoint up` locally and another runs `waypoint up -remote`,
+if one user runs `waypoint up` locally and another runs `waypoint up -local=false`,
```

Comment:
This is actually true even for two local builds! I'd vote we document that case instead: 

```
Therefore, if two users attempt run `waypoint up` at the same time, even if they add the `-local` flag, one of the operations will wait for the other to complete before continuing
```